lib/commands/exam.js

```diff
@@ -358,6 +358,35 @@ module.exports = TestCommand.extend({
       } else {
         this.socket.emit('get-more-tests-response', moduleName);
       }
+
+      const browserId = (() => {
+        const testPage = this.launcher.settings.test_page;
+        return /_browser=\s*(.*)/.exec(testPage)[1];
+      })();
+
+      proto.moduleMap[browserId] = proto.moduleMap[browserId] || [];
+      proto.moduleMap[browserId].push(moduleName);
+    }
+
+    events['after-tests-complete'] = function () {
+      const proto = Object.getPrototypeOf(this);
+      const fileName = 'test-execution-%TIMESTAMP%.json'.replace('%TIMESTAMP%', Date.now());
+
+      if(!proto.finishedBrowser) {
+        proto.finishedBrowser = 1;
+      }
+
+      const testPages = config.testPage;
```

Comment:
Is 'config.testPage' really supposed to allow multiple pages? If so, it should be named config.testPages. If not, the variable should be 'testPage'. In either case, because it's never used except to get the keys for it, maybe you can skip the variable altogether and just do Object.keys(config.testPage) ?